sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -2652,7 +2825,7 @@ private void ValidatePrebuiltForm(RecognizedForm recognizedForm, bool includeFie
         {
             Assert.NotNull(recognizedForm.FormType);
             Assert.IsTrue(recognizedForm.FormTypeConfidence.HasValue);
-            Assert.AreEqual(1.0f, recognizedForm.FormTypeConfidence.Value);
+            Assert.That(recognizedForm.FormTypeConfidence.Value, Is.LessThanOrEqualTo(1.0).Within(0.005));
```

Comment:
We're getting back a confidence of 0.995 in the `StartRecognizeIdDocumentsCanAuthenticateWithTokenCredential`. Do we have any guarantee from the service that it should be 1.0 or is this something we assumed?

Reply:
hmm interesting. So until now, the prebuilt models were not returning a confidence value because if you ask for the specific prebuilt, the service will only use that model. so confidence should be 1.0f.
I think it is worth checking with the service team why this value is 1) returned and 2) why is not 1 ...
not saying it is wrong, but misleading

Reply:
Now that I think about it, it makes sense. The form type is `prebuilt:idDocument:driverLicense`, so the service tried to identify the type of ID Document, something it didn't need to do for other endpoints.

Reply:
Ohhhh this is a great point! wow. yeah, then it makes sense